src/Scratch.as

```diff
@@ -135,6 +136,9 @@ public class Scratch extends Sprite {
 
 	public var logger:Log = new Log(16);
 
+	// Offline Queue
+	private var offlineSystem:OfflineSystem = OfflineSystem.getInstance();
+
```

Comment:
I think it would be best to move this line into the `scratch-flash-offline` repository, along with `OfflineSystem.as`.